### PR TITLE
Add tests for books#search

### PR DESCRIPTION
**Pull request by marcegl91**
### Summary

- Add tests for books#search method using doubles and mocks

### Trello Card

https://trello.com/c/5WD0W8In/39-crear-test-para-el-endpoint-de-book-information

### CODE REVIEW

File: wbooks-api/spec/controllers/api/v1/books_controller_spec.rb [hunk 1]
```diff
@@ -58,4 +58,46 @@
       end
     end
   end
+
+  describe 'GET #search' do
+    book = {
+      isbn: '0385472579',
```
Comment (facucompe) [inline wbooks-api/spec/controllers/api/v1/books_controller_spec.rb]: Create a let with this isbn code

File: wbooks-api/spec/controllers/api/v1/books_controller_spec.rb [hunk 2]
```diff
@@ -58,4 +58,46 @@
...
+      title: 'Zen speaks',
```
Comment (facucompe) [inline wbooks-api/spec/controllers/api/v1/books_controller_spec.rb]: Try using fakers

File: wbooks-api/spec/controllers/api/v1/books_controller_spec.rb [hunk 3]
```diff
@@ -58,4 +58,46 @@
...
+      end
+
+      it 'responds with 200 status' do
+        expect(response).to have_http_status(:ok)
+      end
+    end
+
+    context 'when fetching an invalid book' do
+      before do
+        get :search, params: { isbn: '13483783354345434' }
```
Comment (facucompe) [inline wbooks-api/spec/controllers/api/v1/books_controller_spec.rb]: Same for this, create a let for this code.

**Review by facucompe — DISMISSED** [review]

**Review by floresv — DISMISSED** [review]
LGTM

**Review by floresv — APPROVED** [review]

**Pull request merged**